pkg/migrations/migrations.go

```diff
@@ -0,0 +1,109 @@
+package migrations
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/OpenSlides/openslides-manage-service/pkg/connection"
+	"github.com/OpenSlides/openslides-manage-service/proto"
+	"github.com/spf13/cobra"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/status"
+)
+
+const (
+	// MigrationsHelp contains the short help text for the command.
+	MigrationsHelp = "Wrapper to the OpenSlides backend migration tool for allying migrations to the datastore."
```

Comment:
typo allying

Reply:
I do not understand. Should I write `applying`

Reply:
Yes, i was under the impression that's what you wanted to write. And I also thin It'd make sense